### DISCUSSION TITLE
Why Alpine is not used for bulk actions ?

### DISCUSSION THREAD

**Discussion by Fred7877** [rappasoft/laravel-livewire-tables#601] [Q&A]
Hi Anthony,
I noticed that when we select lines for bulk, there is an Ajax call made by livewire, I thought Alpine would be used for this case, I have Alpine installed though.
On the demo page, it's the same thing, is it normal or a bug?

Regards,

**Comment by rappasoft** [comment]
That's normal. Livewire has to keep track of the values of the checkboxes.
